etc/security/mh_default_org.xml

```diff
@@ -125,10 +126,12 @@
     <sec:intercept-url pattern="/services/sanitize" method="POST" access="ROLE_ADMIN, ROLE_UI_SERVICES_STATUS_EDIT" />
     <sec:intercept-url pattern="/staticfiles" method="POST" access="ROLE_ADMIN, ROLE_UI_THEMES_CREATE, ROLE_UI_THEMES_EDIT" />
     <sec:intercept-url pattern="/admin-ng/acl" method="POST" access="ROLE_ADMIN, ROLE_UI_ACLS_CREATE" />
+    <sec:intercept-url pattern="/admin-ng/event/bulk/conflicts" method="POST" access="ROLE_ADMIN, ROLE_UI_EVENTS_DETAILS_SCHEDULING_VIEW" />
```

Comment:
Where would you need a conflict check when you only have read access to that dialog. Should this maybe be `ROLE_UI_EVENTS_DETAILS_SCHEDULING_EDIT`?

Reply:
It is a read request that uses HTTP POST because the a parameter is a potentially large list of event IDs that could cause problems then using a GET request because of length limitations of query parameters.

I agree that this requires somebody looking at this line to know about the implementation which should not be necessary...

Should I add a comment? I could also just change it to a *_EDIT role.

What do you think?